public/docs/ts/latest/guide/attribute-directives.jade

```diff
@@ -61,17 +61,17 @@ a#write-directive
   :marked
     Read more about [style binding](template-syntax.html#style-binding) on the [Template Syntax](template-syntax.html) page.
 
-    For a simple example, though, this will demonstrate how attribute directives work.
+    For a simple example, though, this demonstrates how attribute directives work.
 
 
 
 :marked
   ### Write the directive code
-  Follow the [setup](setup.html) instructions for creating a new project
+  Follow the [setup](setup.html) instructions for creating a new local project
```

Comment:
I'm not against it entirely. But I don't see that the word "local" helps. If you agree, let's revert. If you don't, leave it.